.github/workflows/pr.yaml

```diff
@@ -0,0 +1,23 @@
+name: Docker Image from PR
+
+on:
+  pull_request:
+    branches:
+      - master
+    types:
+      - assigned
+      - edited
```

Comment:
```suggestion
```

It should just always build on any action.

Reply:
done